added kotlin file extension

alkeryn opened the pull request:
Added the file extension for the kotlin programming language

janlazo commented:
[vim-patch:8.2.0190](https://github.com/vim/vim/commit/ab067a21b9622513ed75f4801b001606eeaf2474) does the same but includes a test. Run `./scripts/vim-patch.sh` to create a new commit for the vim-patch.

Alkeryn commented:
@janlazo i'm the author of that modification ^

alkeryn committed:
vim-patch:8.2.0190: Kotlin files are not recognized

Problem:    Kotlin files are not recognized.
Solution:   Detect Kotlin files. (Alkeryn, closes vim/vim#5560)
https://github.com/vim/vim/commit/ab067a21b9622513ed75f4801b001606eeaf2474

janlazo commented on runtime/filetype.vim.rej:
```diff
@@ -0,0 +1,15 @@
+--- runtime/filetype.vim
```
`*.rej` files should not be committed.

```
-" Dockerfilb; Podman uses the same syntax with name Containerfile
+" Dockerfile; Podman uses the same syntax with name Containerfile
```
suggests that an earlier runtime patch is required but it's fine. These typo changes happen a lot.

Alkeryn replied:
Oh ok gonna fix it wait a sec ^
